lib/index.js

```diff
@@ -353,6 +354,85 @@ const feedbackHandler = async function feedbackHandler(options) {
   logger.info('Thank you!');
 };
 
+async function triggerUpsertHandler(options) {
+  // options = {"_":["triggers","remove","triggerName"],"$0":"bn",...otherOptions}
+  // We have to use the position number here because of lack of good support
+  // for nested commands with positional arguments
+  // consider replacing with `nested-yargs` npm package
+  const triggerName = options._[2];
+  const { sourceUri, target, errorHandler } = options;
+  const triggerOptions = {
+    source: {
+      uri: sourceUri,
+    },
+    target: {
+      funcName: target,
+    },
+  };
+  if (errorHandler) {
+    triggerOptions.errorHandler = {
+      funcName: errorHandler,
+    };
+  }
+  const { status } = await triggers.upsert(triggerName, triggerOptions);
+  logger.info(status);
+}
+
+async function triggerRemoveHandler(options) {
+  // options = {"_":["triggers","remove","triggerName"],"$0":"bn"}
+  // We have to use the position number here because of lack of good support
+  // for nested commands with positional arguments
+  // consider replacing with `nested-yargs` npm package
+  const triggerName = options._[2];
+  const { status } = await triggers.remove(triggerName);
+  logger.info(status);
+}
+
+function formatTriggerSource(source) {
+  if (!source) {
+    return undefined;
+  }
+
+  const { proto, tls, host, port, db, streamName } = source;
+  return `${proto}${tls ? '+tls' : ''}://${host}:${port}/${db}#${streamName}`;
```

Comment:
omit port if 6379, omit db if 0, show password if exists (will be XXX when returned by the server).

Reply:
done